pkg/trace/stats/quantile/converter_test.go

```diff
@@ -77,6 +79,20 @@ func getConvertedSketchQuantiles(t *testing.T, n int, gen func(i int) float64, t
 	return hits, errors
 }
 
+func testUnmarshalDenseSketchFromFile(filename string, zeroCount int, contiguousBinLength int) func(t *testing.T) {
```

Comment:
I really like this approach. Do you plan to add more tests below? Otherwise can we just absorb this into that test and declare it inline? I don't think there's a need for a function declared separately if it's not reused again.

Reply:
yes I'd like to be able to add more with minimal fuss, hence this approach